cmd/zpool/zpool_main.c

```diff
@@ -6912,6 +6936,70 @@ zpool_do_trim(int argc, char **argv)
 	return (error);
 }
 
+/*
+ * Print out detailed error scrub status.
+ */
+static void
+print_err_scrub_status(pool_scan_stat_t *ps)
+{
+	time_t start, end, pause;
+	uint64_t total_secs_left;
+	uint64_t secs_left, mins_left, hours_left, days_left;
+	uint64_t examined, to_be_examined;
+
+	if (ps == NULL || ps->pss_error_scrub_func != POOL_ERRORSCRUB) {
+		return;
+	}
+
+	(void) printf(gettext("  errorscrub: "));
```

Comment:
Current output:
```
errorscrub: error scrub in progress since Mon Aug 12 15:58:11 2019
```
I would just keep this as "scrub:"
```
scrub: error scrub in progress since Mon Aug 12 15:58:11 2019
```